From 8e74a2e223bd40e8cf95ac2580df56ac8500a859 Mon Sep 17 00:00:00 2001
From: Boyuan Yang <byang@debian.org>
Date: Wed, 22 Apr 2020 10:29:39 -0400
Subject: [PATCH] configure.ac: Fix leftover on shell compatibility

Uses "=" instead of "==" in string comparision for POSIX
compatibility.  This is a leftover of previous similar fix.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 105da09..df0220a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -845,7 +845,7 @@ dnl summary log
 echo ""
 echo "please verify that the detected configuration matches your expectations:"
 echo "------------------------------------------------------------------------"
-AS_IF([test "x$USE_NLS" == "xyes"],
+AS_IF([test "x$USE_NLS" = "xyes"],
   [echo    "gettext                $USE_NLS"]
 )
 AS_IF([test "x$win32" != "xno"],
